src/OSPSuite.Core/Domain/Services/PKValuesCalculator.cs

```diff
@@ -77,7 +78,7 @@ private void setMultipleDosingPKValues(ICache<string, double> pk, IReadOnlyList<
 
          //Last Interval
          setCmaxAndTmax(pk, lastInterval, C_max_tDLast_tDEnd, Tmax_tDLast_tDEnd);
-         setValue(pk, Ctrough_tDLast, lastInterval, x => x.CTrough);
+         setValue(pk, Ctrough_tDLast, lastMinusOneInterval, x => x.CTrough);
```

Comment:
If you're looking for a reason to use the word `penultimate` this is it.

Reply:
don't use penultimate if the goal is to communicate clearly. It's a real word with a definition, but many people get it wrong.